WebAssembly/Instructions/Int32TruncateSaturateFloat32Signed.cs

```diff
@@ -0,0 +1,79 @@
+﻿using System.Reflection.Emit;
+using WebAssembly.Runtime;
+using WebAssembly.Runtime.Compilation;
+
+namespace WebAssembly.Instructions
+{
+    /// <summary>
+    /// Truncate (with saturation) a 32-bit float to a signed 32-bit integer.
+    /// </summary>
+    public class Int32TruncateSaturateFloat32Signed : MiscellaneousInstruction
+    {
+        /// <summary>
+        /// Always <see cref="MiscellaneousOpCode.Int32TruncateSaturateFloat32Signed"/>.
+        /// </summary>
+        public override MiscellaneousOpCode MiscellaneousOpCode =>
```

Comment:
This override should always be sealed so that someone inheriting this class can't override it and cause strange behavior.  Might be a good idea to add a unit test to cover all `MiscellaneousInstruction` inheritors.

Reply:
Done - although I'm not sure exactly what you meant about a unit test, so I haven't done that.

Reply:
No worries, I'll take care of that next time I work on this project (probably the coming weekend).